src/project_model/package_dependencies.rs

```diff
@@ -0,0 +1,26 @@
+use cargo_metadata::{Dependency, Package};
+use std::path::PathBuf;
+
+pub struct PackageWithDependencies<'a>(pub &'a Package, pub Vec<PackageDependency>);
+
+pub struct PackageDependency {
+    pub manifest_path: PathBuf,
+    pub _features: Vec<String>,
+    pub _uses_default_features: bool,
+}
+
+impl PackageDependency {
+    pub fn new(dependency: &Dependency, all_packages: &[Package]) -> Self {
+        let manifest_path = &all_packages
+            .iter()
+            .find(|p| p.name == dependency.name)
+            .unwrap()
```

Comment:
czemu to się musi udać?

Reply:
Tak mi się wydawało z początku ale jak dokładniej przeanalizowałem, to wcale nie mam pewności, dlatego poprawiłem.